src/pages/Quote.js

```diff
@@ -0,0 +1,12 @@
+/* eslint-disable */
```

Comment:
- [OPTIONAL] Instead of disabling `eslint` it will be nice to fix the issue, since `linter` are there to help us implement best practices. Please kindly fix the issue.